cmd/antrea-agent/options.go

```diff
@@ -522,8 +522,8 @@ func (o *Options) validateEgressConfig(encapMode config.TrafficEncapModeType) er
 	if !features.DefaultFeatureGate.Enabled(features.Egress) {
 		return nil
 	}
-	if encapMode != config.TrafficEncapModeEncap {
-		klog.InfoS("The Egress feature gate is enabled, but it won't work because it is only applicable to the encap mode")
+	if encapMode != config.TrafficEncapModeEncap && encapMode != config.TrafficEncapModeHybrid {
```

Comment:
nit: we already have a helper function for this condition

```
encapMode.SupportsEncap()
```

Reply:
Done